src/components/NavBar/NavBar.js

```diff
@@ -1,11 +1,20 @@
 import React from 'react';
 import './NavBar.css';
+import { Link } from "react-router-dom";
 
-const NavBar = () => {
+const NavBar = (props) => {
   return (
     <section className="navbar-container">
-      <h3 className="vrad-header">VRAD</h3>
-      <h2 className="page-header">Listings</h2>
+      <h3 className="vrad-header"><span className="v">V</span>RAD</h3>
+      <h2 className="page-header">{props.title}</h2>
+      <section className="user-about-container">
+        <p className="welcome-user">Welcome, {props.userInfo.name}!</p>
+        <Link className="logout-link" to="/">
+          <button onClick={() => props.signOut()} className="logout-button">Sign Out</button>
```

Comment:
Link routes back to home, onClick calls the signOut method from app to update userInfo state and isLoggedIn state.